Monika After Story/game/script-apologies.rpy

```diff
@@ -0,0 +1,250 @@
+#Create an apology db for storing our times
+default persistent._mas_apology_time_db = {}
```

Comment:
can you add documentation here that explains what exactly is the structure of stuff stored in here?

like how I tend to comment:
```python
# each element is a tuple of the following format:
# [0] - contains x
# [1] - contains y
```